src/js/main.js

```diff
@@ -67,12 +67,12 @@ var SupportKit = Marionette.Object.extend({
         }
 
 
-        if(/lebo|awle|pide|obo|rawli/i.test(navigator.userAgent)) {
+        if (/lebo|awle|pide|obo|rawli/i.test(navigator.userAgent)) {
```

Comment:
Do we not want to bring back what Chris had? Do we know if this method works correctly with Google's bot?